index.html

```diff
@@ -0,0 +1,166 @@
+                  /></a>
+                </li>
+                <li class="menu-top-item">
+                  <a href="#"
+                    ><img src="resources/images/icon_twitter.png" alt=""
+                  /></a>
+                </li>
+                <li class="menu-top-item">
+                  <a href="#">English</a>
+                </li>
+              </ul>
+            </div>
+            <div class="menu-main">
+              <div class="menu-img">
+                <img
+                  class="menu-button"
+                  src="resources/images/icon-menu.svg"
+                  alt="menu icon"
+                />
+                <img
+                  class="menu-logo"
+                  src="resources/images/logo.png"
+                  alt="logo"
+                />
+                <div>&nbsp;</div>
+              </div>
+              <ul id="menu-list" class="menu-list hidden">
+                <li class="menu-item"><a href="index.html">Home</a></li>
+                <li class="menu-item"><a href="about.html">About</a></li>
+                <li class="menu-item">
+                  <a href="mailto:sonaban.art@gmail.com">Contact</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <section id="intro" class="section intro">
+        <h2 class="event-welcome text-gradient">¡Bienvenidos, tuiteros!</h2>
+        <h1 class="event-title text-gradient">EXPO tuiteratura<br />2022</h1>
+        <p class="event-message">
+          <em>Expo tuiteratura 2022</em> es una iniciativa de Ana Baños,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>, arropada
+          con entusiasmo por 200 noveles hilanderos, todos jóvenes
+          universitarios, estudiantes de 4to, 7mo y 9no semestre de la facultad
+          de Psicología - UANL.
+        </p>
+        <h4 class="event-dates">25 de abril al<br />15 de mayo de 2022</h4>
+        <h4 class="event-organizer">
+          <a href="mailto:sonaban.art@gmail.com">sonaban.art@gmail.com</a>,
+          <a href="https://twitter.com/sonabana_art">@sonabana_art</a>
+        </h4>
+      </section>
+
+      <section id="program" class="section program">
+        <h2 class="section-title">Programa</h2>
+        <div class="separator"></div>
+        <div class="program-container">
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_01.png" alt="icon-1" />
+            </div>
+            <div class="program-title">Conferencia</div>
+            <div class="program-description">
+              Excucha a los expositores de varios países compartiendo sus
+              experiencias.
+            </div>
+          </div>
+          <div class="program-item">
+            <div class="program-icon">
+              <img src="resources/images/program_icon_02.png" alt="icon-1" />
+            </div>
```

Comment:
- [ ] Great job so far, for mobile devices, in order to stick to the design make sure the spacing(margin, padding) looks like the design.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163457923-28288d52-6d0d-4597-ac6a-6e684800f650.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163458038-ba28ad35-55a1-471f-98f8-ef987ed7c1f7.png)